docs/source/resources/scyllaclusters/storage-resizing.md

```diff
@@ -0,0 +1,99 @@
+# Resizing storage in ScyllaCluster
+
+Due to limitations of the underlying StatefulSet, ScyllaClusters don't allow storage changes. The following procedure describes how to adjust the storage manually.
```

Comment:
One of the requirements is that StorageClass used supports resising. It should be mentioned here and explained how to verify that.